src/applications/healthcare/questionnaire/tests/e2e/01.questionnaire.demographics.cypress.spec.js

```diff
@@ -0,0 +1,71 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+
+import basicUser from './fixtures/users/user-basic.json';
+
+describe('healthcare questionnaire -- demographics', () => {
+  before(() => {
+    cy.server();
+    cy.fixture(
+      '../../src/applications/healthcare/questionnaire/tests/e2e/fixtures/mocks/feature-toggles.enabled.json',
+    ).then(async features => {
+      cy.route('GET', '/v0/feature_toggles*', features);
+      cy.login(basicUser);
+      window.localStorage.setItem(
+        'DISMISSED_ANNOUNCEMENTS',
+        JSON.stringify(['single-sign-on-intro']),
+      );
+    });
+    cy.visit('/healthcare/questionnaire/demographics');
+  });
+  it('all default phone numbers', () => {
+    cy.get('[data-test-id="homePhone"]', {
```

Comment:
We have [Cypress Testing Library](https://testing-library.com/docs/cypress-testing-library/intro), and the [`cy.findByTestId` query](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid) would be appropriate throughout this test.